hyundai_kia_connect_api/Vehicle.py

```diff
@@ -143,6 +151,24 @@ def last_service_distance(self, value):
         self._last_service_distance_unit = value[1]
         self._last_service_distance = value[0]
 
+    @property
+    def location_latitude(self):
+        return self._location_latitude
+
+    @property
+    def location_longitude(self):
+        return self._location_longitude
+
+    @property
+    def location(self):
+        return self._location_longitude, self._location_latitude
+    
+    @location.setter
+    def location(self, latitude, longitude):
+        self._location_latitude = latitude
+        self._location_longitude = longitude
+        self._location_last_set_time = datetime.datetime.now(pytz)
```

Comment:
do we have this as a response of an API, last updated of location? we can expect API to pass this?

Reply:
I haven't seen the API respond with it.  Date will be for reducing calls since this maxes out regions and we have logic in some that ensures this isn't called too often.